packages/nodes-base/nodes/Telegram/Telegram.node.ts

```diff
@@ -33,6 +48,26 @@ export class Telegram implements INodeType {
 				required: true,
 			},
 		],
+		webhooks: [
+			{
+				name: 'default',
```

Comment:
Will these always be the same for all nodes? Could be worth moving it to the util

Reply:
thanks, done